src/pages/starter-kit/integration/create-integration.md

```diff
@@ -311,7 +311,7 @@ Check your App in the Developer Console to confirm the registrations were create
 
 <InlineAlert variant="info" slots="text"/>
 
-The following steps are not required when using Adobe I/O Events version `1.6.0` or later. The onboarding script will configure the Adobe Commerce instance automatically.
+The following steps are not required when using Adobe I/O Events version `1.12.0` or later. The onboarding script will configure the Adobe Commerce instance automatically.
```

Comment:
Since everyone must use 1.12.0 or later, I think we can delete this whole procedure and note